menuconfig.py

```diff
@@ -299,6 +299,141 @@
     "brightpurple": curses.COLOR_MAGENTA + 8,
 }
 
+def rgb_to_6cube(rgb):
```

Comment:
Not sure if good idea, but maybe these functions could take three parameters instead.